cmd/root.go

```diff
@@ -10,12 +12,15 @@ import (
 	"github.com/spf13/viper"
 )
 
+var configFileUsed bool
+
 var rootCmd = &cobra.Command{
-	Use:     "preflight",
-	Short:   "Preflight Red Hat certification prep tool.",
-	Long:    "A utility that allows you to pre-test your bundles, operators, and container before submitting for Red Hat Certification.",
-	Version: version.Version.String(),
-	Args:    cobra.MinimumNArgs(1),
+	Use:              "preflight",
+	Short:            "Preflight Red Hat certification prep tool.",
+	Long:             "A utility that allows you to pre-test your bundles, operators, and container before submitting for Red Hat Certification.",
+	Version:          version.Version.String(),
+	Args:             cobra.MinimumNArgs(1),
+	PersistentPreRun: preRunConfig,
```

Comment:
Maybe my other two comments aren't valid, and this takes are of all the sub commands.

Reply:
Correct.